File: Modul-2/PRAK204/PRAK204-2310817210010-Harry-Pratama-Yunus.c
#include <stdio.h>
#include <math.h>
int main(){
    float jari_jari,tinggi_bejana;
    float pi=3.142857;
    scanf("%f %f",&jari_jari,&tinggi_bejana);
    float Volume=pi*jari_jari*jari_jari*tinggi_bejana;
    float luas = 2 * pi * jari_jari * (jari_jari + tinggi_bejana);
    float keliling = 2 * pi * jari_jari;
    printf("Volume = %.2f\n",Volume);
    printf("luas = %.2f\n",luas);
    printf("Keliling = %.2f\n",keliling);
    return 0;
}

File: Modul-2/PRAK204/PRAK204-2310817210010-Harry-Pratama-Yunus.py
Angka=input().split()
if len(Angka)==1:
  jari_jari=float(Angka[0])
  tinggi_bejana=float(input())
elif len(Angka)==2:
  jari_jari,tinggi_bejana=map(float,Angka)
pi=3.142857
Volume=pi*jari_jari*jari_jari*tinggi_bejana
luas = 2 * pi * jari_jari * (jari_jari + tinggi_bejana)
keliling = 2 * pi * jari_jari
print("Volume = {:.2f}".format(Volume))
print("luas = {:.2f}".format(luas))
print("Keliling = {:.2f}".format(keliling))